native-image-logging-examples/README.md

```diff
@@ -0,0 +1,92 @@
+# Native Image Logging Configuration Examples
+
+This repository contains the code for a demo application for [GraalVM](http://graalvm.org).
+
+### Prerequisites
+* [GraalVM](http://graalvm.org)
+* [Native Image](https://www.graalvm.org/docs/reference-manual/native-image/)
+
+## Preparation
+
+1. [Download GraalVM](https://www.graalvm.org/downloads/), unzip the archive, export the GraalVM home directory as the `$GRAALVM_HOME` and add `$GRAALVM_HOME/bin` to the `PATH` environment variable.
```

Comment:
Replace step 1 with just this line of text:

1. Download and install the latest GraalVM JDK with Native Image using [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader):
  ```
  bash <(curl -sL https://get.graalvm.org/jdk)
  ```